numba/typing/npydecl.py

```diff
@@ -168,6 +168,11 @@ class NumpyRulesArrayOperator(Numpy_rules_ufunc):
          '<': "less",
         '<=': "less_equal",
         '!=': "not_equal",
+        '+=': "in_add",
+        '-=': "in_subtract",
+        '*=': "in_multiply",
+        '/=': "in_true_divide",
+       '//=': "in_floor_divide",
```

Comment:
There's a number of them missing: `%=`, `**=`, `<<=`, `>>=`, `&=`, `|=`, `^=`.
Also, rather than have a special name ("in_XXX"), why not a separate subclass? "numpy.in_add" doesn't exist.